gustaf/spline/ffd.py

```diff
@@ -165,8 +141,7 @@ def spline(self):
         --------
         self._spline: Spline
         """
-        self._logd("Returning copy of current spline.")
-        return self._spline.copy() if self._spline is not None else None
+        return self._spline if self._spline is not None else None
```

Comment:
```suggestion
        return self._spline
```
wouldn't this be enough, since `_spline` is initialized as `None`?